rpi_csdt_community/views.py.orig

```diff
@@ -1,40 +0,0 @@
-"""Display the id, name, description, and url for the demo."""
-from django.contrib.auth.forms import AuthenticationForm
-from django.http import HttpResponse
-from django.shortcuts import render
-from django_comments.views.comments import post_comment
-
-from project_share.models import Project
-
-
-def comment_post_wrapper(request):
-<<<<<<< HEAD
-    """Clean the request to prevent form spoofing."""
-    if request.user.is_authenticated():
-        if not (request.user.get_full_name() == request.POST['name'] or
-                request.user.email == request.POST['email']):
-=======
-    # Clean the request to prevent form spoofing
-    if request.user.is_authenticated:
-        if not (request.user.get_full_name() == request.POST['name'] or \
-               request.user.email == request.POST['email']):
->>>>>>> 9ca58e084f44c2e60af221b2fbe6f91849fe25be
-            return HttpResponse("Error 403: You're an evil hacker")
-        return post_comment(request)
-    return HttpResponse("Error 403: You're an evil hacker")
-
-
-def home(request):
-    """Get the 10 most popular projects *dead*, Get the 10 newest."""
-    projects_newest = Project.approved_projects().all().select_related("screenshot").order_by('-id')[:10]
-    projects_newest = [project for project in projects_newest]
-    return render(request, 'home.html', {
-        "form": AuthenticationForm(),
-        'projects_popular': projects_newest,
-        'projects_newest': projects_newest
-    })
-
-
-def return_true(req):
-    """Unused and depricated."""
-    return True
```

Comment:
Please Delete all your .DS_Store files and add .DS_Store to the .gitignore file.